addons/stock/static/src/fields/stock_statusbar_field.js

```diff
@@ -0,0 +1,29 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { statusBarField, StatusBarField } from "@web/views/fields/statusbar/statusbar_field";
+
+export class StockStatusBarField extends StatusBarField {
+    static template = "stock.StockStatusBarField";
```

Comment:
```suggestion
export class StockPickingLockedStatusBarField extends StatusBarField {
    static template = "stock.PickingLockedStatusBarField";
```
`Sounds better and is clearly related to the picking and its status.`